src/compiler/test-file/formats/raw/prepare-command-object.js

```diff
@@ -0,0 +1,23 @@
+// NOTE: TestCafe Studio adds additional fields to the command object in RAW tests.
```

Comment:
We write new files with TypeScript to gradually move our codebase to TypeScript.